msmbuilder/featurizer/featurizer.py

```diff
@@ -629,44 +682,66 @@ def partial_transform(self, traj):
+                - resids: unique residue ids (0-indexed)
+                - featurizer: Alpha Angle
+                - featuregroup: the type of dihedral angle and whether sin or
+                  cos has been applied.
+        """
+        feature_descs = []
         # fill in the atom indices using just the first frame
         self.partial_transform(traj[0])
+        top = traj.topology
         if self.atom_indices is not None:
-            aind = self.atom_indices
-            n = len(aind)
-            resSeq = [(np.unique([traj.top.atom(j).residue.resSeq for j in i]))
-                      for i in aind]
-            resid = [(np.unique([traj.top.atom(j).residue.index for j in i]))
-                     for i in aind]
-            resnames = [[traj.topology.residue(j).name for j in i]
-                        for i in resid]
-            bigclass = ["dihedral"] * n
-            smallclass = ["alpha"] * n
-
+            aind_tuples = self.atom_indices
+            resseqs = []
+            resids = []
+            resnames = []
+            for ainds in aind_tuples:
+                resid = set(top.atom(ai).residue.index for ai in ainds)
+                resids += [list(resid)]
+                resseqs += [[top.residue(ri).resSeq for ri in resid]]
+                resnames += [[top.residue(ri).name for ri in resid]]
+
+            zippy = zip(aind_tuples, resseqs, resids, resnames)
             if self.sincos:
-                aind = list(aind) * 2
-                resnames = resnames * 2
-                resSeq = resSeq * 2
-                resid = resid * 2
-                otherInfo = (["sin"] * n) + (["cos"] * n)
-                bigclass = bigclass * 2
-                smallclass = smallclass * 2
+                zippy = itertools.product(['sin', 'cos'], zippy)
             else:
-                otherInfo = ["nosincos"] * n
-
-            for i in range(len(resnames)):
-                d_i = dict(resname=resnames[i], atomind=aind[i],
-                           resSeq=resSeq[i], resid=resid[i],
-                           otherInfo=otherInfo[i], bigclass=bigclass[i],
-                           smallclass=smallclass[i])
-                x.append(d_i)
-
-            return x
+                zippy = itertools.product(['nosincos'], zippy)
+
+            for sincos, info in zippy:
+                ainds, resseq, resid, resname = info
+                feature_descs += [dict(
+                    resnames=resname,
+                    atominds=ainds,
+                    resseqs=resseq,
+                    resids=resid,
+                    featurizer="AlphaAngle",
+                    featuregroup="{}".format(sincos),
+                )]
+
+            return feature_descs
         else:
-            raise UserWarning("Cannot describe features for trajectories with fewer than 4 alpha carbon\
+            raise UserWarning("Cannot describe features for trajectories with "
+                              "fewer than 4 alpha carbon\
```

Comment:
Don't use `\` line continuation in strings, it will include all the spaces! Use auto-string concat
```python
func("a long string "
        "that spans multiple lines.")